sound/soc/codecs/rt715-sdca.c

```diff
@@ -286,7 +288,22 @@ static int rt715_sdca_put_volsw(struct snd_kcontrol *kcontrol,
 		if (err < 0)
 			return err;
 	}
-
+#if IS_ENABLED(CONFIG_DELL_PRIVACY)
+	/* Privacy LED Trigger State Changed by muted/unmute switch */
+	if (mc->invert) {
+		val0 = ucontrol->value.integer.value[0];
+		val1 = ucontrol->value.integer.value[1];
+		if (val0 == 1 && val1 == 1) {
+			rt715->micmute_led = LED_OFF;
+			ledtrig_audio_set(LED_AUDIO_MICMUTE,
```

Comment:
config DELL_PRIVACY
        tristate "Dell Hardware Privacy Support"
        depends on ACPI
        depends on ACPI_WMI
        depends on INPUT
        depends on DELL_LAPTOP
        depends on LEDS_TRIGGER_AUDIO

when privacy driver enabled, it can enable the LEDS_TRIGGER_AUDIO . so we do not need check that again.

Reply:
> doesn't this need `#if IS_REACHABLE(CONFIG_LEDS_TRIGGER_AUDIO)` similar to sound/soc/sof/control.c? See commit [9899a7a](https://github.com/thesofproject/linux/commit/9899a7a869935c4c93247b290ac7a70e0deab202) ("ASoC: SOF: Fix randbuild error")

LEDS_TRIGGER_AUDIO